Example/CardParts/DummyCardController.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  DummyCardController.swift
+//  CardParts_Example
+//
+//  Created by Tumer, Deniz on 5/23/18.
+//  Copyright © 2018 CocoaPods. All rights reserved.
+//
+
+import Foundation
+import CardParts
+
+class DummyCardController: CardPartsViewController {
```

Comment:
Do we need this still? Or was this just for testing performance?

Reply:
it was just testing performance I can remove it if we don't need it anymore